### PR TITLE
Added clouds to earth

### PR DESCRIPTION
**Pull request by StefanDeYoung**
Created earth.py to handle the logic for drawing the clouds as a sphere around the earth. It does read from earth.atmopheric_thickness to determine the size of the cloud sphere.

I haven't figured out how to make the clouds rotate faster than the earth.

### CODE REVIEW

**Comment by pmelanson** [comment]
Let me know when this is ready to merge and I'll give it a final look

**Comment by StefanDeYoung** [comment]
@pmelanson should be good now.

File: orbitx/physics/calc.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def orb_speed(habitat: Entity,
     """The orbital speed of an astronomical body or object.
     Equation referenced from https://en.wikipedia.org/wiki/Orbital_speed"""
     return math.sqrt(
-        (reference.mass**2 * common.G) /
+        (reference.mass ** 2 * common.G) /
```
Comment (pmelanson) [inline orbitx/physics/calc.py:52]: Can you keep this formatted with autopep8? idk if pycharm has a convenient plugin for it.

If there's an easy way to have autopep8 automatically format, that'll keep the code style consistent and formatting changes like this away.

But it's just a small thing so if it's not easy, don't bother.

Comment (pmelanson) [reply orbitx/physics/calc.py:52]: Same for all the other lines in this file

Comment (StefanDeYoung) [reply orbitx/physics/calc.py:52]: Weird! Pycharm kept giving me formatting suggestions that I accepted. I must have accepted a formatting change for the whole file o.0. I'll look into it.

**Pull request merged**